### DISCUSSION TITLE
Any way to move via keyboard?

### DISCUSSION THREAD

**Discussion by zarakon** [diasurgical/DevilutionX#4091] [Q&A] [answered]
Since it's possible to move using a gamepad, is there any way to do the same with keyboard (e.g. WASD)?

**Comment by StephenCWills** [comment, marked as answer]
There is a build option for this, but you'll have to compile it yourself.
https://github.com/diasurgical/devilutionX/blob/master/Source/controls/README.md

**Reply by AJenbo** [reply]
As a side note it will probably be avalible from the options menu in 1.5.0

**Reply by zarakon** [reply]
Okay, thanks!
